packages/app/script-utils/src/vault/syncEnvWithVault.ts

```diff
@@ -43,9 +43,14 @@ const upsertEnvValue = (
  *
  * @param envVars Record of variable key-values
  * @param file Path to the .env file
+ * @param createEmpty Whether to create empty file even if no variables are passed
  */
-export const updateEnvFile = (envVars: Record<string, string>, file: string) => {
-  if (Object.entries(envVars).length === 0) {
+export const updateEnvFile = (
+  envVars: Record<string, string>,
+  file: string,
+  createEmpty = false,
+) => {
```

Comment:
So this function is just replicated wholesale in the autopilot scripts apart from this one addition which seems silly. it's not a breaking change as without the new param it has the same behaviour, but now we can use this one instead of just a copied and pasted version.